src/index.js

```diff
@@ -191,8 +225,15 @@ export default (function create(defaults) {
 		});
 	}
 
-	redaxios.CancelToken = self.AbortController || Object;
+ /**
+  * @public
+  * @type {AbortController}
+  */
+  redaxios.CancelToken = /** @type {any} */ (self).AbortController || Object;
```

Comment:
this is TypeScript dom types issue as `AbortController` is not defined on `Window` interface